content/en/docs/concepts/policy/pod-security-policy.md

```diff
@@ -599,8 +599,10 @@ documentation](/docs/tutorials/clusters/apparmor/#podsecuritypolicy-annotations)
 
 ### Seccomp
 
-The use of seccomp profiles in pods can be controlled via annotations on the
-PodSecurityPolicy. Seccomp is an alpha feature in Kubernetes.
+You can use the `seccompProfile` field in the `securityContext` of Pods or
+containers to [control use of seccomp
+profiles](../../tutorials/clusters/seccomp.md), and you can use
+PodSecurityPolicies to enforce how these fields are applied.
```

Comment:
Let's mention that the PSP enforces both: the annotation and the new fields like described here: https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/20190717-seccomp-ga.md#podsecuritypolicy-enforcement

Reply:
Great point, thanks @saschagrunert!

Reply:
Updated :+1: